config/opensearch_dashboards.yml

```diff
@@ -237,6 +237,8 @@
 
 # Set the value of this setting to true to enable multiple data source feature.
 #data_source.enabled: false
+# Set the value of this setting to true to enable default cluster in data source feature.
+#data_source.defaultCluster: false
```

Comment:
Default should be true from the `decideClient` logic?

Reply:
Good catch, will update.

Reply:
Updated, resolving comments